MonoGame.Framework/Desktop/OpenTKGamePlatform.cs

```diff
@@ -88,13 +88,11 @@ class OpenTKGamePlatform : GamePlatform
 		private OpenALSoundController soundControllerInstance = null;
         // stored the current screen state, so we can check if it has changed.
         private bool isCurrentlyFullScreen = false;
-        private Toolkit toolkit;
         private int isExiting; // int, so we can use Interlocked.Increment
         
 		public OpenTKGamePlatform(Game game)
             : base(game)
         {
-            toolkit = Toolkit.Init();
```

Comment:
For us it does nothing.

Also from what I read Toolkit is only used if we are combining OpenTK with something else, which we are not.

http://www.opentk.com/files/doc/class_open_t_k_1_1_toolkit.html
> You must call this method if you are combining OpenTK with a third-party windowing toolkit (e.g. GTK#).